source/event.c

```diff
@@ -190,6 +190,7 @@ void event_core_write(arm_core_event_s *event)
     bool added = false;
     ns_list_foreach(arm_core_event_s, event_tmp, &event_queue_active) {
         // note enum ordering means we're checking if event_tmp is LOWER priority than event
+
```

Comment:
You shouldn't be touching this file at all. Squash your commits together, and make sure this file is untouched.